### ISSUE TITLE
Black screen between the Splash Screen and first Flutter widget

### ISSUE THREAD

**Issue by nickmeinhold** [nickmeinhold/silicon_beach_member#1]
Relevant Issues: 
- [First start on Android flashes: starts with black and then displays the app (#5373) · Issue #22007 · flutter/flutter](https://github.com/flutter/flutter/issues/22007)
- [Black screen flashes between the LaunchImage and app's first screen on iOS · Issue #20827 · flutter/flutter](https://github.com/flutter/flutter/issues/20827) 

Several people report that the issue is only there in Debug mode and goes away in Release mode 

Flutter docs for the add-to-app feature mention a custom splash screen that allows you to control how that View transitions to Flutter after Flutter renders its first frame: 
[Adding a splash screen and launch screen to an Android app - Flutter](https://flutter.dev/docs/development/add-to-app/android/add-splash-screen#creating-a-custom-splashscreen)

My brief testing supports that the issue doesn't occur in Release mode but leaving the issue open so we can 
- review if anyone has seen the issue in release mode 
- do some further testing to decide if it's definitely not an issue

**Comment by hb0nes** [comment]
Yes, I've seen this in release mode. I have it on me and my girlfriends phone. 
It's SUPER ugly.

**Comment by nickmeinhold** [comment]
@hb0nes thanks for the info, where have you seen it exactly?  Do you have a minimal repro to add to the tracking issues?

**Comment by hb0nes** [comment]
No, I don't have time to build one right now, but it appears to be gone by following this comment:
https://github.com/flutter/flutter/issues/39494#issuecomment-565627914

You could try the following
```dart
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await Future.delayed(Duration(seconds: 2));
  runApp(MyApp());
}
```

This is my actual code now, after following the comment I linked above.
```dart
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await AppRepository().loadApp();
  setupLocator();
  RendererBinding.instance.initPersistentFrameCallback();
  runApp(MyApp());
}
```

**Comment by nickmeinhold** [comment]
Great! Thanks for the info!
